docs/source-app/core_api/lightning_app/communication_content.rst

```diff
@@ -0,0 +1,174 @@
+
+----
+
+*************************************************
+Communication from LightningWork to LightningFlow
+*************************************************
+
+LightningFlow (Flow) can access their children's LightningWork (Work) state.
+
+When a running Work attribute gets updated inside its method (separate process locally or remote machine), the app re-executes Flow's run method once it receives the state update from the Work.
+
+Here's an example to better understand communication from Work to Flow.
+
+The ``WorkCounter`` increments a counter until 1 million and the ``Flow`` prints the work counter.
+
+As the Work is running its own process, its state changes are sent to the Flow which contains the latest value of the counter.
+
+.. code-block:: python
+
+    import lightning as L
+
+
+    class WorkCounter(L.LightningWork):
+        def __init__(self):
+            super().__init__(parallel=True)
+            self.counter = 0
+
+        def run(self):
+            for _ in range(int(10e6)):
+                self.counter += 1
+
+
+    class Flow(L.LightningFlow):
+        def __init__(self):
+            super().__init__()
+            self.w = WorkCounter()
+
+        def run(self):
+            self.w.run()
+            print(self.w.counter)
+
+
+    app = L.LightningApp(Flow())
+
+
+A delta sent from the Work to the Flow looks like this:
+
+.. code-block:: python
+
+    {"values_changed": {"root['works']['w']['vars']['counter']": {"new_value": 425}}}
+
+Here is the associated illustration:
+
+.. figure:: https://pl-flash-data.s3.amazonaws.com/assets_lightning/deltas.gif
+    :alt: Mechanism showing how delta are sent.
+    :width: 100 %
+
+Here's another example that is slightly different. Here we define a Flow and Work, where the Work increments a counter indefinitely and the Flow prints its state which contains the Work.
+
+You can easily check the state of your entire app as follows:
+
+.. literalinclude:: ../../core_api/lightning_app/app.py
+
+Run the app with:
+
+.. code-block:: bash
+
+    lightning run app docs/source-app/core_api/lightning_app/app.py
+
+And here's the output you get when running the App using the **Lightning CLI**:
+
+.. code-block:: console
+
+	INFO: Your app has started. View it in your browser: http://127.0.0.1:7501/view
+	State: {'works': {'w': {'vars': {'counter': 1}}}}
+	State: {'works': {'w': {'vars': {'counter': 2}}}}
+	State: {'works': {'w': {'vars': {'counter': 3}}}}
+	State: {'works': {'w': {'vars': {'counter': 3}}}}
+	State: {'works': {'w': {'vars': {'counter': 4}}}}
+	...
+
+----
+
+*************************************************
+Communication from LightningFlow to LightningFlow
+*************************************************
+
+Every LightningFlow (Flow) can access the state of any of its children Flow's.
+
+Here's an example to better understand communication from Flow to Flow.
+
+.. code-block:: python
+
+    import lightning as L
+
+----
```

Comment:
@tchaton The example seems to be missing here?

Reply:
Done !